docs/source/getting_started/dataset_creation/dataset_creation_gcloud_bucket.rst

```diff
@@ -1,6 +1,189 @@
+    ) || (
+        resource.type == 'storage.googleapis.com/Object' &&
+        resource.name.startsWith("projects/_/buckets/lightly-datalake/objects/projects/wild-animals")
+    )
+
+For more information, head to the `IAM conditions
+<https://cloud.google.com/storage/docs/access-control/iam#conditions>`_.
+The first part of the condition adds listing rights to the whole bucket,
+as they can only be handled on the bucket level. The second part adds object-level
+access rights (i.e. read and create) for all objects in the bucket `lightly-datalake`
+whose name starts with `projects/wild-animals`.
+
+.. figure:: images_gcloud_bucket/screenshot_gcloud_create_service.jpg
+    :align: center
+    :alt: Google Cloud Service Account
+    :width: 60%
+
+
+
+- Click on `Done` to create the service account.
+- You can change the roles of the service account later in the
+  `IAM <https://console.cloud.google.com/iam-admin/iam>`_.
+
+6. Navigate to `APIs -> Credentials <https://console.cloud.google.com/apis/credentials>`_
+   again if you are not already there.
+
+- Find the just created user in the list of all service accounts.
+- Click on the user and navigate to the `keys` tab.
+- Click on `Add key` and create a new private key in JSON Format.
+  It will download the corresponding key file.
+
+.. figure:: images_gcloud_bucket/screenshot_gcloud_service_account_key_creation.jpg
+    :align: center
+    :alt: Google Cloud Service Account Key Creation
+    :width: 60%
+
+
+
+
+Configuring a Lightly dataset to access the Google Cloud Storage
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Create and configure a dataset
+
+1. `Create a new dataset <https://app.lightly.ai/dataset/create>`_ in Lightly
+2. Edit your dataset and select `Google Cloud Storage` as your datasource
+
+.. figure:: images_gcloud_bucket/screenshot_gcloud_create_dataset.jpg
+    :align: center
+    :alt: Configure google cloud bucket datasource in Lightly Platform
+    :width: 60%
+
+
+3. As the resource path, enter the full URI to your resource eg. `gs://lightly-datalake/projects/wild-animals`
+4. Enter the Google Project ID you wrote down in the first step.
+5. Click on `Select Credentials File` to add the key file you downloaded in the previous step.
+6. The thumbnail suffix depends on the option you chose in the first step
+
+- Option A: You want Lightly to create the thumbnail for you.
+  Then choose the naming scheme to your liking.
+- Option B: You have already generated thumbnails in your bucket.
+  Then choose the thumbnail suffix such that it reflects you naming scheme.
+- Option C: You don't want to use thumbnails.
+  Then leave the thumbnail suffix undefined/empty.
+
+
+
+6. Press save and ensure that at least the lights for List and Read turn green.
+If you added permissions for writing to use option A, this light should also turn green.
+
+7. Now you should be on the dataset creation page again.
```

Comment:
That's not true, you are still on the `edit dataset` screen. I would just leave this bullet point away

Reply:
Or write that you can now exit the settings by clicking the X

Reply:
Good point, I changed it to `7. After closing the pop-up by clicking the X, you should be on the dataset creation page again.`.